examples/ai-lab-example/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "cd ../../ && pnpm build --filter ai-lab && pnpm build --filter @ai-lab/web-example && cd examples/ai-lab-example && pnpm storybook",
+    "start": "pnpm storybook",
```

Comment:
Since we don't need to build let's not do it anymore. Reduces storybook startup time.